static/app/components/replays/replayPlatformIcon.tsx

```diff
@@ -1,58 +0,0 @@
-import {useTheme} from '@emotion/react';
-import styled from '@emotion/styled';
-import {PlatformIcon as BasePlatformIcon} from 'platformicons';
-
-import {Tooltip} from 'sentry/components/core/tooltip';
-import CountTooltipContent from 'sentry/components/replays/countTooltipContent';
-import {t} from 'sentry/locale';
-import {space} from 'sentry/styles/space';
-import {generatePlatformIconName} from 'sentry/utils/replays/generatePlatformIconName';
-import type {Theme} from 'sentry/utils/theme';
-
-type Props = {
-  name: string;
-  version: undefined | string;
-  className?: string;
-  showTooltip?: boolean;
-  showVersion?: boolean;
```

Comment:
turns out we didn't use the props `showTooltip`, which meant that `showVersion` was unused too... the whole component simplified to be:
```
const icon = generatePlatformIconName(name, version);
return <BasePlatformIcon platform={icon} size={iconSize} style={iconStyle(theme)} />;
```

so i just cleaned it up and out.